CONTRIBUTING.md

```diff
@@ -0,0 +1,99 @@
+- Put a blank line between the title, body and footer
+- Wrap the commit body at 72 characters
+- Try to write the commit body as **bullets** ( optional )
+- Use imperative in title ( **Add** instead of **Added**, for example )
+- Describe what and why, but not how ( the purpose, not the implementation )
+- Footer contains issue number, if applicable ( #78 )
+
+> For more information about the 50/72 characters rule, please visit [this link.](https://dev.to/noelworden/improving-your-commit-message-with-the-50-72-rule-3g79#:~:text=The%2050%2F72%20Rule%20is,pulled%20out%20of%20a%20hat.)
+
+Your commits should follow the [conventional commits specification.](https://www.conventionalcommits.org/en/v1.0.0/#summary)
+
+Please be mindful of this conventionn when writing your commits. CI jobs will check if your commits follow this convention, since these commits might later be used to create changelog using parsers that understand this convention specifically.
+
+## When to make an Issue, and what's its goal ?
+An issue should be opened when new functional needs are defined, a new bug is discovered, or a documentation resource is going to be created. You **must** open an issue before opening the corresponding branch.  An issue should not be technical, and should describe what the end user will get. You open an issue when either one of the following condition is true :
+- You want to describe what your team is currently working on : this issue is meant to be merge from your team's branch to the main branch
+- You want to describe features that you are currently working on : this issue is meant to be merge from a feature branch to your team's branch
+
+## How to make an issue ?
+
+When creating a new issue, you'll be presented with 2 possibles templates. Please choose the one that fits best what you're trying to do.
+You can then follow the form.
+
+The title of the issue should contains what it is about in the least amount of words possible ( container health check )
+For example : `Container health check`
+
+To describe what you will add to the program, you need to provide a tasks list. For example :
+### Tasks list: 
+- [x] Create cluster command
+- [ ] Delete cluster command
+
+You should tag each tasks with its or their corresponding feature branch.
+
+After creating your issue, if it is a feature issue, you need to create the branch. To the right of the issue in GitHub, you'll find, under `Development`, a button allowing you to `Create a branch`, linked to the issue. You can then click on `create branch`. All work related to this feature issue should be pushed in this branch.
+
+## When to make a PR, and what's its goal ?
+There are two kinds of PR you can make : either a feature branch you want to merge into your team's branch, or your team's branch you want to merge into the main branch. You should create the PR as soon as you have created the branch for your issue, though you should mark it as a draft PR. 
+
```

Comment:
Sorry for sounding like a broken record, but here as well: That's an internal flow.